routes/user.routes.js

```diff
@@ -0,0 +1,9 @@
+const express = require("express");
+const router = express.Router();
+const user = require("../controllers/user.controller");
+
+router.route("/adduser").post(user.createUser);
+
+router.route("/userlogin").post(user.userLogin);
+
+module.exports = router;
```

Comment:
When testing your api in postman. the base route will be localhost:{your port}/api/v1/{resource}.